lib/matobj1.gd

```diff
@@ -264,3 +264,111 @@ DeclareCategory( "IsMatrixObj", IsMatrixOrMatrixObj );
 ##
 DeclareCategory( "IsRowListMatrix", IsMatrixObj );
 
+
```

Comment:
Why do two files exist and what is supposed to be in one or the other?

Reply:
This is explained at the top of these two files: one is read earlier in the GAP loading process than the other, to deal with various dependencies between declarations in multiple files.